doc/network-zones.md

```diff
@@ -2,66 +2,151 @@
 discourse: 12033,13128
 ---
 
-# Network Zones configuration
-Network zones are used to hold DNS records for LXD networks.
+# How to configure network zones
 
-Each network can be related to up to 3 zones for:
+```{note}
+Network zones are available for the {ref}`OVN network <network-ovn>` and the {ref}`Bridge network <network-bridge>`.
+```
 
- - Forward DNS records
- - IPv4 reverse DNS records
- - IPv6 reverse DNS records
+Network zones can be used to hold DNS records for LXD networks.
 
-This is controlled through `dns.zone.forward`, `dns.zone.reverse.ipv4`
-and `dns.zone.reverse.ipv6` in network configuration. LXD will then be
-automatically managing forward and reverse records for all instances,
-network gateways and downstream network ports.
+If you are operating a LXD cluster with multiple instances across a large set of networks, you can use network zones to automatically maintain valid forward and reverse records for all your instances.
+Such DNS records are important not only for ease of access to the instances, but also to avoid hosted services getting flagged as potential spam due to lacking reverse DNS records.
 
-To enable the built-in DNS server, `core.dns_address` must be set in the
-server configuration.
+Each network can be related to up to three zones for:
 
-The built-in DNS server only supports zone transfers through AXFR, it
-cannot be directly queried for DNS records. This means that this feature
-expects the use of an external DNS server (bind9, nsd, ...) which will
-transfer the entire zone from LXD, refresh it upon expiry and provide
-authoritative answers to DNS requests.
+- Forward DNS records
+- IPv4 reverse DNS records
+- IPv6 reverse DNS records
 
-Authentication for zone transfer is configured on a per-zone basis with
-peers defined in zone configuration and a combination of IP address
-matching and TSIG key based authentication.
+LXD will then automatically manage forward and reverse records for all instances, network gateways and downstream network ports and serve those zones for zone transfer to the operator’s production DNS servers.
 
-Zones belong to projects and are tied to the `networks` features of projects.
+So for example, if you configure a zone for forward DNS records for `lxd.example.net` for your network, it is used to resolve DNS names for all instances in the network (`<instance_name>.lxd.example.net`).
+If you configure a zone for IPv4 reverse DNS records for `30.192.10.in-addr.arpa` for a network using `10.192.30.0/24`, it is used for reverse DNS lookup for, for example, `10.192.30.100`.
+
+## Enable the built-in DNS server
+
+To make use of network zones, you must enable the built-in DNS server.
+
+To do so, set the `core.dns_address` configuration option (see {doc}`server`) to the address where the DNS server should be available.
```

Comment:
"Should" or "will"? This is the build-in DNS server and you specify where it should be available after the configuration - it's not an external server that you set up and provide the address here. Correct?

Reply:
Well "will" ideally, assuming there isn't anything else listening on that address.

How about:

> To do so, set the `core.dns_address` configuration option (see {doc}`server`) to a local address on the LXD server that the DNS server will listen on.

Then we've been specific that the address should be local to each LXD server (as in a cluster it may be different on each cluster member). This might be worth mentioning explicity below too.